nemo/core/classes/modelPT.py

```diff
@@ -222,3 +245,16 @@ def val_dataloader(self):
     # def test_dataloader(self):
     #     if self._test_dl is not None:
     #         return self._test_dl
+
+    def _update_hyperparameters(self, *args, **kwargs):
```

Comment:
@okuchaiev Should I make this private or public (its used by all subclasses to log any scalar value into either Tensorboard or other Loggers.